web/client/components/catalog/RecordGrid.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+
+const {Grid, Row, Col} = require('react-bootstrap');
+
+const RecordItem = require('./RecordItem');
```

Comment:
Can make it, but we need to include RecordGrid as a part of the recordgrid to make it really configurable from config.js files.